### DISCUSSION TITLE
patch generation

### DISCUSSION THREAD

**Discussion by yousri-daldoul** [Arkady23/V380-Web#37] [General]
Hello and thank you for sharing this project.

I used this patch httpd_V380E2_C2_9090909_v20210620.zip to install your server and it worked perfectly.

Then I tried to create my own patch to install another program on the camera. So I modified the file updatepatch/69d9e5d488199b358ca38beebc8f9313.patch and I added some new lines.

Unfortunately, the modified patch did not work and does not upgrade the camera anymore.

Could you tell me how did you create this patch: httpd_V380E2_C2_9090909_v20210620.zip

Thank you

**Comment by daoudeddy** [comment]
Follow this [link](https://github.com/bcaller/v380-ipcam-firmware-patch)

**Reply by yousri-daldoul** [reply]
Ok, thank you. I tried it before posting the comment and it did not work. Then I added -h V380E2_C2 and it worked.
